app/crud.py

```diff
@@ -44,38 +51,38 @@ def create_step(db: Session,
         settings = config
 
     db_step = models.Step(**step.dict())
-    db_step.status = 'prelaunch'
+    db_step.status = "prelaunch"
     db_step.script_version = settings["version_number"]
 
-    dir_head = settings['folders']['working_dir']
-    dir_tail = '/' + settings['cc_batch']
-    further_params = "-L " + settings['runtime_configurations']['log_level']
+    dir_head = str(Path(settings["folders"]["working_dir"]).expanduser())
+    dir_tail = "/" + settings["cc_batch"]
+    further_params = "-L " + settings["runtime_configurations"]["log_level"]
 
     # Some steps can run on only one process and thus have no processes param.
-    # These steps are marked by 'no_p_param: True' in the config.yaml.
-    if not settings["scripts"][db_step.step_name].pop('no_p_param', False):
-        further_params += ' -P ' \
-                          + str(config['runtime_configurations']['processes'])
+    # These steps are marked by "no_p_param: True" in the config.yaml.
+    if not settings["scripts"][db_step.step_name].pop("no_p_param", False):
+        further_params += " -P " \
+                          + str(config["runtime_configurations"]["processes"])
 
     # Let's go over and process the parameters:
     for key, value in settings["scripts"][db_step.step_name].items():
-        if key == 'script_file':

```

Comment:
I wouldn't mind double quotes, but we use single quotes everywhere else in the code...

Reply:
Not anymore! :)

In the app/ I overwhelmingly used double quotes. So I unified it to use double quotes everywhere within the app/

Reply:
OK, at least we'll have easy authorship attribution :laughing: